experimental/oak_baremetal_app_crosvm/layout.ld

```diff
@@ -50,11 +52,19 @@ SECTIONS {
         bss_start = .;
         *(.bss .bss.*)
         bss_size = . - bss_start;
-    }
+    } : ram
 
     /* Stack grows down, so stack_start is the upper address in memory. */
     .stack (NOLOAD) : ALIGN(4K) {
         . += 512K;
-    }
+    } : ram
     stack_start = .;
+
+    .cpuid (NOLOAD) : ALIGN(2M) {
```

Comment:
I don't think the 2M alignment adds value. Even if we decide to us 2M pages, the AMD Secure Processor that will populate these will always use 4K granularity for adding these special pages.

Reply:
My idea was that if we're going for 2M page size, this will allow us to map the cpuid page and secrets page separately (otherwise the two 4K pages would be on the same 2M page). Not sure how much value it would add in the grand scheme, though.

Reply:
I can't think of a scenario where we would want different page table attributes or permissions for the two sections (e.g. we want both to be encrypted,  read-only and non-executable). But, I don't think it makes a big difference for now either way so it you think there might be value in future it is fine to leave as is.